data/supportserver/named/db.rpz

```diff
@@ -0,0 +1,10 @@
+$TTL 60
+@            IN    SOA  localhost. root.localhost.  (
+                      2   ; serial 
+                      3H  ; refresh 
+                      1H  ; retry 
+                      1W  ; expiry 
+                      1H) ; minimum 
+              IN    NS    localhost.
+
+updates.suse.com        A    11.22.33.44
```

Comment:
Can we use the variable EDGECAST instead of the `11.22.33.44`

Reply:
sure, that's just for testing that it really changes things